### ISSUE TITLE
Interfacial area given by SNOW is wrong

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/porespy#263]
In `add_phase_interconnections` method, the array that is preallocated to store interfacial areas is of dtype `int`, and therefore the interfacial areas most of the times end up being zero (since voxel_size is often below 1 -> area is truncated to 0 because of `int` dtype).

`dtype=float` should be passed to the `zeros_like` method below to fix the problem.
```python
        for j in phases_num:
            if j > i:
                pi_pj_sa = np.zeros_like(label)
                loc6 = np.logical_and(conns2 >= num[j - 1], conns2 < num[j])
                pi_pj_conns = loc1 * loc6
                net['throat.{}_{}'.format(al[i], al[j])] = pi_pj_conns
```

**Comment by Zohaib-Atiq** [comment]
@ma-sadeghi I have fixed this issue already. Once snow parallel will be merged this will be fine.

**Comment by jgostick** [comment]
This was fixed by #265
